style/ruby/.rubocop.yml

```diff
@@ -467,6 +467,9 @@ Style/InitialIndentation:
   Description: >-
     Checks the indentation of the first non-blank non-comment line in a file.
   Enabled: false
+  
+Style/SymbolArray:
```

Comment:
@awead updated the cop with a description and based on the code and added a link to the location of the documentation